src/server/routes/launchpad.js

```diff
@@ -14,6 +15,9 @@ router.use('/launchpad/snaps', json());
 router.post('/launchpad/snaps', newSnap);
 router.get('/launchpad/snaps', findSnap);
 
+router.use('/launchpad/snaps/list', json());
+router.get('/launchpad/snaps/list', findSnaps);
```

Comment:
I'd prefer `GET /launchpad/snaps` in the long-term, but it'd be a bit confusing since they have different return types.  Let's stick with this for now and adjust later; I suspect that `GET /launchpad/snaps` can be decommissioned once the new authz workflow is in place, at which point we'll have an easier time rearranging it.